src/script/components/MessagesList/Message/ContentMessage/index.tsx

```diff
@@ -199,13 +221,15 @@ const ContentMessageComponent: React.FC<ContentMessageProps> = ({
               className="message-body-like-icon like-button message-show-on-hover"
               message={message}
               onLike={onLike}
+              focusConversation={focusConversation}
             />
           </div>
         )}
 
         <div className="message-body-actions">
-          {menuEntries.length > 0 && (
+          {menuEntries.length && (
```

Comment:
the previous check is actually better unless you want to see "0" rendered in the DOM